### ISSUE TITLE
About the type definition of interpolate and the use of extrapolate parameter？

### ISSUE THREAD

**Issue by zejunking** [pmndrs/react-spring#1105]
https://github.com/react-spring/react-spring/blob/8af3b2aaf52e91b4f361cba91a9affae643f854e/src/interpolate.ts#L83

When it's used, this is the only type definition, and extrapolate cannot be used. Will it be obsolete or integrated by default?
```
// "version": "8.0.27",
export function interpolate(
  parent: number[],
  config: (...args: number[]) => any
): any
```

**Comment by aleclarson** [comment]
Pretty sure this is fixed in v9 (#985)

**Comment by zejunking** [comment]
> Pretty sure this is fixed in v9 (#985)

Thanks
